packages/graphback-codegen-schema/src/SchemaCRUDPlugin.ts

```diff
@@ -726,7 +728,15 @@ export class SchemaCRUDPlugin extends GraphbackPlugin {
         });
       }
 
-      return context[dataLoaderName].load(parent[relationship.relationForeignKey])
+      const relationForeignKey = parent[relationship.relationForeignKey];
+
+      // eslint-disable-next-line no-null/no-null
+      if (relationForeignKey === undefined || relationForeignKey === null) {
```

Comment:
Catches both:

```suggestion
      if (!relationForeignKey) {
```

Reply:
I wanted to cater for cases where `relationForeignKey` is either `0` or `''`.